go/private/rules/sdk.bzl

```diff
@@ -69,6 +70,13 @@ go_sdk = rule(
             doc = ("Pre-compiled .a files for the standard library, " +
                    "built for the execution platform"),
         ),
+        "exports": attr.label_list(
+            # allow_files is not set to [".x"] because that wouldn't allow
```

Comment:
Which error are you seeing in this case? The only ones I can find shouldn't be shown when there are no files.